app.organicmaps.desktop.yml

```diff
@@ -23,10 +23,10 @@ modules:
     sources:
       - type: git
         url: https://github.com/organicmaps/organicmaps
-        tag: 2023.06.04-13-android
-        commit: acc7c0547db4285dd8841ae7f98811268e38d908
+        branch: master
+        commit: be5a0aa2991377eedd61f19ec3853332b51b114b
```

Comment:
this will break as soon as there is a new commit.

Reply:
#37